internal/presentation/gui.go

```diff
@@ -302,6 +302,15 @@ func (g gameUi) drawCurrentPlayer(boardOffset int) error {
 	}
 
 	currentPlayerBox.Clear()
+
+	if g.board.State == model.GameStateEnd {
+		fmt.Fprintf(currentPlayerBox, `
+
+         GAME OVER
+`)
```

Comment:
Is this indentation intended ? 
Or it's for display purpose ?

Reply:
It is for display purpose.